### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#463] [Esami & Esoneri M-Z]
## Traccia

<img width="1065" alt="image" src="https://github.com/user-attachments/assets/8d00c43b-f218-4508-b49f-3e84d952bce4">

**Comment by rimaout** [comment]
Vero, $x+y$ è una relazione di equivalenza. Ciò è dovuto al fatto che per ottenere un numero basta sommare due qualsiasi numeri pari o due qualsiasi numeri dispari. Questa proprietà si verifica grazie alle seguenti caratteristiche:

- **Riflessiva**: la somma di un numero con se stesso è sempre pari, poiché $x+x = 2x$.
- **Simmetrica**: la somma di due numeri è commutativa, quindi $x+y = y+x$. Di conseguenza, se $x+y$ è pari, lo è anche $y+x$.
- **Transitiva**: se sia $x+y$ che $y+z$ sono pari, allora $x$, $y$ e $z$ sono tutti pari o tutti dispari. Pertanto, anche $x+z$ è pari.

Le classi di equivalenza sono due, ovvero la classe dei numeri pari e la classe dei numeri dispari. Ciò è dovuto al fatto che:
- Tutti i numeri pari sono in relazione tra loro e non sono mai in relazione con un numero dispari.
- Tutti i numeri dispari sono in relazione tra loro e non sono mai in relazione con un numero pari.

Quindi l'insieme quoziente è Q = {[1], [2]}, dove [1] rappresenta la classe di equivalenza dei numeri dispari e [2] quella dei numeri pari.

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/509025b9-ea44-4256-be4d-882e19dcd4d4)
